```
Error running mint automation
+ mkdir -p 12269-9e0b900/gopath/src/github.com/minio
+ git -C 12269-9e0b900/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 12269-9e0b900/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 12269-9e0b900/gopath/src/github.com/minio/minio fetch --quiet minio pull/12269/head:pr12269
+ git -C 12269-9e0b900/gopath/src/github.com/minio/minio checkout --quiet pr12269
+ GO111MODULE=on
+ GOPATH=/home/mint-auto/mint-auto/12269-9e0b900/gopath
+ make -C 12269-9e0b900/gopath/src/github.com/minio/minio --quiet
go: downloading github.com/minio/cli v1.22.0
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/minio/madmin-go v1.0.2
go: downloading github.com/minio/minio-go/v7 v7.0.11-0.20210302210017-6ae69c73ce78
go: downloading cloud.google.com/go v0.57.0
go: downloading github.com/Azure/azure-pipeline-go v0.2.2
go: downloading github.com/Azure/azure-storage-blob-go v0.10.0
go: downloading github.com/colinmarc/hdfs/v2 v2.2.0
go: downloading github.com/json-iterator/go v1.1.11
go: downloading github.com/jcmturner/gokrb5/v8 v8.4.2
go: downloading google.golang.org/api v0.25.0
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/minio/sha256-simd v1.0.0
go: downloading github.com/mattn/go-isatty v0.0.12
go: downloading github.com/minio/highwayhash v1.0.2
go: downloading github.com/mattn/go-colorable v0.1.8
go: downloading github.com/fatih/color v1.10.0
go: downloading github.com/secure-io/sio-go v0.3.1
go: downloading golang.org/x/crypto v0.0.0-20210421170649-83a5a9bb288b
go: downloading github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: downloading golang.org/x/sys v0.0.0-20210420072515-93ed5bcd2bfe
go: downloading github.com/rjeczalik/notify v0.9.2
go: downloading github.com/montanaflynn/stats v0.5.0
go: downloading github.com/minio/sio v0.3.0
go: downloading github.com/bcicen/jstream v1.0.1
go: downloading github.com/klauspost/readahead v1.3.1
go: downloading github.com/klauspost/reedsolomon v1.9.11
go: downloading github.com/google/uuid v1.1.2
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/cheggaaa/pb v1.0.29
go: downloading github.com/djherbis/atime v1.0.0
go: downloading github.com/gorilla/handlers v1.5.1
go: downloading github.com/klauspost/compress v1.12.2
go: downloading github.com/klauspost/pgzip v1.2.5
go: downloading github.com/VividCortex/ewma v1.1.1
go: downloading github.com/gorilla/mux v1.8.0
go: downloading github.com/miekg/dns v1.1.35
go: downloading github.com/minio/selfupdate v0.3.1
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/minio/rpc v1.0.0
go: downloading github.com/pierrec/lz4 v2.6.0+incompatible
go: downloading github.com/philhofer/fwd v1.1.1
go: downloading github.com/prometheus/client_golang v1.8.0
go: downloading github.com/prometheus/procfs v0.6.0
go: downloading github.com/rs/cors v1.7.0
go: downloading github.com/shirou/gopsutil/v3 v3.21.3
go: downloading github.com/tinylib/msgp v1.1.3
go: downloading github.com/willf/bloom v2.0.3+incompatible
go: downloading github.com/prometheus/client_model v0.2.0
go: downloading go.etcd.io/etcd v0.0.0-20201125193152-8a03d2e9614b
go: downloading golang.org/x/net v0.0.0-20210421230115-4e50805a0758
go: downloading github.com/coredns/coredns v1.4.0
go: downloading go.uber.org/zap v1.14.1
go: downloading github.com/beevik/ntp v0.3.0
go: downloading github.com/fatih/structs v1.1.0
go: downloading gopkg.in/yaml.v2 v2.3.0
go: downloading github.com/minio/simdjson-go v0.2.1
go: downloading github.com/go-ldap/ldap/v3 v3.2.4
go: downloading github.com/dswarbrick/smart v0.0.0-20190505152634-909a45200d6d
go: downloading github.com/Shopify/sarama v1.27.2
go: downloading github.com/eclipse/paho.mqtt.golang v1.3.0
go: downloading github.com/go-sql-driver/mysql v1.5.0
go: downloading github.com/gomodule/redigo v2.0.0+incompatible
go: downloading github.com/lib/pq v1.9.0
go: downloading github.com/nats-io/nats.go v1.10.0
go: downloading github.com/nats-io/stan.go v0.8.3
go: downloading github.com/nsqio/go-nsq v1.0.8
go: downloading github.com/olivere/elastic/v7 v7.0.22
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/streadway/amqp v1.0.0
go: downloading github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: downloading github.com/ncw/directio v1.0.5
go: downloading github.com/minio/parquet-go v1.0.0
go: downloading github.com/minio/csvparser v1.0.0
go: downloading github.com/alecthomas/participle v0.2.1
go: downloading github.com/mattn/go-ieproxy v0.0.1
go: downloading github.com/spaolacci/murmur3 v1.1.0
go: downloading github.com/willf/bitset v1.1.11
go: downloading github.com/klauspost/cpuid v1.3.1
go: downloading github.com/mattn/go-runewidth v0.0.4
go: downloading github.com/golang/protobuf v1.4.3
go: downloading github.com/felixge/httpsnoop v1.0.1
go: downloading github.com/Azure/go-ntlmssp v0.0.0-20200615164410-66371956d46c
go: downloading github.com/go-asn1-ber/asn1-ber v1.5.1
go: downloading github.com/golang/snappy v0.0.3
go: downloading github.com/xdg/stringprep v1.0.0
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/gorilla/websocket v1.4.2
go: downloading github.com/nats-io/nuid v1.0.1
go: downloading github.com/gogo/protobuf v1.3.2
go: downloading golang.org/x/text v0.3.6
go: downloading go.uber.org/multierr v1.5.0
go: downloading go.uber.org/atomic v1.6.0
go: downloading git.apache.org/thrift.git v0.13.0
go: downloading github.com/tidwall/sjson v1.1.6
go: downloading github.com/tidwall/gjson v1.7.5
go: downloading github.com/nats-io/nkeys v0.2.0
go: downloading github.com/nats-io/jwt v1.1.0
go: downloading github.com/jcmturner/gofork v1.0.0
go: downloading github.com/hashicorp/go-uuid v1.0.2
go: downloading github.com/jcmturner/dnsutils/v2 v2.0.0
go: downloading github.com/jcmturner/aescts/v2 v2.0.0
go: downloading github.com/jcmturner/rpc/v2 v2.0.3
go: downloading github.com/prometheus/common v0.14.0
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/tklauser/go-sysconf v0.3.4
go: downloading cloud.google.com/go/storage v1.8.0
go: downloading github.com/minio/argon2 v1.0.0
go: downloading github.com/tidwall/pretty v1.1.0
go: downloading github.com/tidwall/match v1.0.3
go: downloading google.golang.org/protobuf v1.24.0
go: downloading github.com/tklauser/numcpus v0.2.1
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: downloading gopkg.in/jcmturner/gokrb5.v7 v7.5.0
go: downloading github.com/eapache/queue v1.1.0
go: downloading github.com/eapache/go-resiliency v1.2.0
go: downloading github.com/rcrowley/go-metrics v0.0.0-20200313005456-10cdbea86bc0
go: downloading gopkg.in/ini.v1 v1.57.0
go: downloading github.com/minio/md5-simd v1.1.1
go: downloading github.com/rs/xid v1.2.1
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.1
go: downloading github.com/klauspost/cpuid/v2 v2.0.4
go: downloading github.com/jcmturner/goidentity/v6 v6.0.1
go: downloading google.golang.org/genproto v0.0.0-20200527145253-8367513e4ece
go: downloading github.com/googleapis/gax-go/v2 v2.0.5
go: downloading google.golang.org/grpc v1.29.1
go: downloading go.opencensus.io v0.22.5
go: downloading golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
go: downloading github.com/mailru/easyjson v0.7.6
go: downloading github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
go: downloading github.com/josharian/intern v1.0.0
go: downloading github.com/coreos/go-semver v0.3.0
go: downloading github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f
go: downloading github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
go: downloading gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: downloading gopkg.in/jcmturner/aescts.v1 v1.0.1
go: downloading gopkg.in/jcmturner/rpc.v1 v1.1.0
# github.com/minio/minio/cmd
cmd/admin-bucket-handlers.go:222:6: json redeclared in this block
	previous declaration at cmd/admin-bucket-handlers.go:158:6
cmd/admin-handlers-users.go:579:6: json redeclared in this block
	previous declaration at cmd/admin-handlers-users.go:495:6
cmd/admin-handlers-users.go:767:6: json redeclared in this block
	previous declaration at cmd/admin-handlers-users.go:753:6
cmd/admin-handlers-users.go:1062:6: json redeclared in this block
	previous declaration at cmd/admin-handlers-users.go:1030:6
make: *** [build] Error 2

```

--- a/cmd/admin-bucket-handlers.go
+++ b/cmd/admin-bucket-handlers.go
@@ -24,6 +24,7 @@ import (
 	"net/http"
 
 	"github.com/gorilla/mux"
+	jsoniter "github.com/json-iterator/go"
 	"github.com/minio/madmin-go"
 	"github.com/minio/minio/cmd/logger"
 	iampolicy "github.com/minio/minio/pkg/iam/policy"
@@ -154,6 +155,7 @@ func (a adminAPIHandlers) SetRemoteTargetHandler(w http.ResponseWriter, r *http.
 		return
 	}
 	var target madmin.BucketTarget
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	if err = json.Unmarshal(reqBytes, &target); err != nil {
 		writeErrorResponseJSON(ctx, w, errorCodes.ToAPIErrWithErr(ErrAdminConfigBadJSON, err), r.URL)
 		return

--- a/cmd/bucket-targets.go
+++ b/cmd/bucket-targets.go
@@ -21,11 +21,11 @@ import (
 	"context"
 	"crypto/sha256"
 	"encoding/hex"
-	"encoding/json"
 	"net/http"
 	"sync"
 	"time"
 
+	jsoniter "github.com/json-iterator/go"
 	"github.com/minio/madmin-go"
 	minio "github.com/minio/minio-go/v7"
 	miniogo "github.com/minio/minio-go/v7"
@@ -385,6 +385,7 @@ func parseBucketTargetConfig(bucket string, cdata, cmetadata []byte) (*madmin.Bu
 		return nil, nil
 	}
 	data = cdata
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	if len(cmetadata) != 0 {
 		if err := json.Unmarshal(cmetadata, &meta); err != nil {
 			return nil, err

--- a/cmd/config/crypto.go
+++ b/cmd/config/crypto.go
@@ -21,11 +21,11 @@ import (
 	"bytes"
 	"crypto/rand"
 	"encoding/binary"
-	"encoding/json"
 	"errors"
 	"fmt"
 	"io"
 
+	jsoniter "github.com/json-iterator/go"
 	"github.com/minio/minio/pkg/fips"
 	"github.com/minio/minio/pkg/kms"
 	"github.com/secure-io/sio-go"
@@ -88,6 +88,7 @@ func Encrypt(KMS kms.KMS, plaintext io.Reader, context kms.Context) (io.Reader, 
 		header [5]byte
 		buffer bytes.Buffer
 	)
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	metadata, err := json.Marshal(encryptedObject{
 		KeyID:     key.KeyID,
 		KMSKey:    key.Ciphertext,
@@ -139,6 +140,7 @@ func Decrypt(KMS kms.KMS, ciphertext io.Reader, context kms.Context) (io.Reader,
 	if _, err := io.ReadFull(ciphertext, metadataBuffer); err != nil {
 		return nil, err
 	}
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	if err := json.Unmarshal(metadataBuffer, &metadata); err != nil {
 		return nil, err
 	}

--- a/cmd/data-usage.go
+++ b/cmd/data-usage.go
@@ -20,7 +20,6 @@ package cmd
 import (
 	"bytes"
 	"context"
-	"encoding/json"
 	"net/http"
 
 	jsoniter "github.com/json-iterator/go"
@@ -41,6 +40,7 @@ const (
 // storeDataUsageInBackend will store all objects sent on the gui channel until closed.
 func storeDataUsageInBackend(ctx context.Context, objAPI ObjectLayer, dui <-chan madmin.DataUsageInfo) {
 	for dataUsageInfo := range dui {
+		var json = jsoniter.ConfigCompatibleWithStandardLibrary
 		dataUsageJSON, err := json.Marshal(dataUsageInfo)
 		if err != nil {
 			logger.LogIf(ctx, err)

--- a/cmd/iam-etcd-store.go
+++ b/cmd/iam-etcd-store.go
@@ -29,6 +29,7 @@ import (
 	"unicode/utf8"
 
 	jwtgo "github.com/dgrijalva/jwt-go"
+	jsoniter "github.com/json-iterator/go"
 	"github.com/minio/madmin-go"
 	"github.com/minio/minio-go/v7/pkg/set"
 	"github.com/minio/minio/cmd/config"
@@ -124,6 +125,7 @@ func getIAMConfig(item interface{}, data []byte, itemPath string) error {
 			}
 		}
 	}
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	return json.Unmarshal(data, item)
 }
 

--- a/cmd/iam-object-store.go
+++ b/cmd/iam-object-store.go
@@ -28,6 +28,7 @@ import (
 	"time"
 	"unicode/utf8"
 
+	jsoniter "github.com/json-iterator/go"
 	"github.com/minio/madmin-go"
 	"github.com/minio/minio/cmd/config"
 	"github.com/minio/minio/cmd/logger"
@@ -244,6 +245,7 @@ func (iamOS *IAMObjectStore) loadIAMConfig(ctx context.Context, item interface{}
 			}
 		}
 	}
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	return json.Unmarshal(data, item)
 }
 

--- a/cmd/jwt/parser.go
+++ b/cmd/jwt/parser.go
@@ -26,7 +26,6 @@ import (
 	"crypto"
 	"crypto/hmac"
 	"encoding/base64"
-	"encoding/json"
 	"fmt"
 	"strings"
 	"sync"
@@ -188,6 +187,7 @@ func (c *MapClaims) Map() map[string]interface{} {
 
 // MarshalJSON marshals the MapClaims struct
 func (c *MapClaims) MarshalJSON() ([]byte, error) {
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	return json.Marshal(c.MapClaims)
 }
 

--- a/cmd/tier-handlers.go
+++ b/cmd/tier-handlers.go
@@ -23,6 +23,7 @@ import (
 	"net/http"
 
 	"github.com/gorilla/mux"
+	jsoniter "github.com/json-iterator/go"
 	"github.com/minio/madmin-go"
 	"github.com/minio/minio/cmd/logger"
 	iampolicy "github.com/minio/minio/pkg/iam/policy"
@@ -85,6 +86,7 @@ func (api adminAPIHandlers) AddTierHandler(w http.ResponseWriter, r *http.Reques
 	}
 
 	var cfg madmin.TierConfig
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	if err := json.Unmarshal(reqBytes, &cfg); err != nil {
 		writeErrorResponseJSON(ctx, w, toAdminAPIErr(ctx, err), r.URL)
 		return
@@ -164,6 +166,7 @@ func (api adminAPIHandlers) EditTierHandler(w http.ResponseWriter, r *http.Reque
 	}
 
 	var creds madmin.TierCreds
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	if err := json.Unmarshal(reqBytes, &creds); err != nil {
 		writeErrorResponseJSON(ctx, w, toAdminAPIErr(ctx, err), r.URL)
 		return

